deps/amqp_client/src/amqp_connection.erl

```diff
@@ -153,12 +157,20 @@ start(AmqpParams) ->
 %% user specified connection name.
 start(AmqpParams, ConnName) when ConnName == undefined; is_binary(ConnName) ->
     ensure_started(),
+    Obfuscate   = fun(Password) -> credentials_obfuscation:encrypt(Password) end,
+    ToPropLists = fun(Props)    -> rabbit_data_coercion:to_proplist(Props) end,
```

Comment:
This says `ToPropLists` but in fact it takes a single value (a map or a proplist) and returns a single list. So perhaps the name should reflect that: `ToProplist`.

Reply:
Done.

Reply:
what is the point of storing function references in variables anyway?

Reply:
what i'm trying to say - instead of two lines for bootstrapping AmqpParams this has 6 lines which are likely slower (depending on how smart compiler is)

Reply:
> what is the point of storing function references in variables anyway?

For the sake of code formatting, @ikavgo: with `credentials_obfuscation:encrypt/1` & `rabbit_data_coercion:to_proplist/1` into the case, the statements go over eighty characters.

Would you like me to change it? I didn't know there are compiling constraints.

Reply:
we have enough lines over 80 :-) I think in this case it will be fine too.

Reply:
:slightly_smiling_face: Fine, then; @ikavgo. There you are.